admin/class-create-theme.php

```diff
@@ -30,6 +31,8 @@ public function __construct() {
 		add_action( 'admin_menu', array( $this, 'create_admin_menu' ) );
 		add_action( 'admin_init', array( $this, 'blockbase_save_theme' ) );
 		add_action( 'enqueue_block_editor_assets', array( $this, 'create_block_theme_enqueue' ) );
+
+		$this->theme = wp_get_theme();
```

Comment:
Does this PR include changes for https://github.com/WordPress/create-block-theme/issues/379? Or are these changes needed for this PR too?

Reply:
Thanks for catching that! I was using those to verify and they are not needed; I've dropped them from the PR!

Reply:
@mikachan Added these changes back in order to get checks passing 😅 See, they're already working :)